### ISSUE TITLE
fix: accessibility on InternalDropdown component

### ISSUE THREAD

**Issue by TahimiLeonBravo** [nexxtway/react-rainbow#1769]
## Requirements
- [ ] It should follow the accessibility standards defined on w3c
https://www.w3.org/TR/wai-aria-practices/#Listbox
- [ ] It should make accessible the InternalDropdown and the Option component
- [ ] It should be tested all the accessibility with jest.

### Tests Specs
- [ ] add specs for testing here
